### PR TITLE
AWSLinux params as RedHat-2.yml, create users prior databases

### PR DESCRIPTION
**Pull request by gusto2**
Installing on AWS Linux2 the ansible recognizes the linux type as "Redhat-2", we created a copy of the RedHat-7 config

When creating databases with owner assigned, it is necessary the user/role already exists

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -19,5 +19,5 @@
     enabled: "{{ postgresql_service_enabled }}"
 
 # Configure PostgreSQL.
-- import_tasks: databases.yml
 - import_tasks: users.yml
+- import_tasks: databases.yml
```
Comment (lhoss) [inline tasks/main.yml]: I'ld undo this change in this PR
and get the change in through (long time open) PR https://github.com/geerlingguy/ansible-role-postgresql/pull/31 , which it would be nice to get merged!

File: tasks/configure.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@
     group: "{{ postgresql_group }}"
     mode: 0600
   notify: restart postgresql
-  when: postgresql_hba_entries | bool
+  when: postgresql_hba_entries is defined
```
Comment (lhoss) [inline tasks/configure.yml]: this is buggy !
here's what you want 👍 : 
```
   when: postgresql_hba_entries | default(false) | bool
```

**Comment by geerlingguy** [comment]
A few of these changes were in other PRs that have already been merged, and for the users before database change, that will be done in #31, likely very soon.

Please open a new PR based off master and split the PR into individual changes so it's easier to review and accept only certain changes.
